shell/platform/android/io/flutter/embedding/engine/FlutterOverlaySurface.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine;
+
+import android.view.Surface;
+
+public class FlutterOverlaySurface {
+  private final Surface surface;
+  private final long id;
+
+  public FlutterOverlaySurface(long id, Surface surface) {
```

Comment:
Same comment about the annotation `public FlutterOverlaySurface(long id, @NonNull Surface surface)`.  `id` doesn't need one because it's a primitive type.